File: Assignment1/VB.NET/WindowsApp1/WindowsApp1/Form1.vb
Public Class Form1
    Public Sub New()
        InitializeComponent()
        Timer1.Interval = 3

    End Sub

    Private r As Rectangle
    Private g As Graphics
    Private x As Integer = 0
    Private y As Integer = 0


    Private Sub MyFristGUI_Load(ByVal sender As Object, ByVal e As EventArgs)
    End Sub




    Private Sub Timer1_Tick(sender As Object, e As EventArgs) Handles Timer1.Tick
        Dim rand = New Random()
        x = rand.[Next](280, 660)
        y = rand.[Next](220, 330)
        g.Clear(BackColor)
        g.DrawRectangle(New Pen(Brushes.Cornsilk, 6), r)
        g.DrawCircle(New Pen(Brushes.Cornsilk, 2), x, y, 20)
        g.FillCircle(Brushes.Cornsilk, x, y, 20)
    End Sub

    Private Sub Button1_Click(sender As Object, e As EventArgs) Handles Button1.Click
        Me.RichTextBox1.Text = "button pressed"
    End Sub

    Private Sub Button2_Click(sender As Object, e As EventArgs) Handles Button2.Click
        r = New Rectangle(260, 200, 400, 150)
        g = Me.CreateGraphics()
        Timer1.Interval = 3



        Timer1.Start()
            Me.Button2.Visible = False
            Me.Button3.Text = "STOP"
        Me.Button3.Visible = True
    End Sub

    Private Sub Button3_Click(sender As Object, e As EventArgs) Handles Button3.Click
        Timer1.Stop()
        g.Clear(BackColor)
        Me.Button3.Visible = False
        Me.Button2.Visible = True
    End Sub
End Class

File: Assignment1/C#/New/MyVeryfirstCSharpProgram/Form1.cs
using MyVeryfirstCSharpProgram.Properties;
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace MyVeryfirstCSharpProgram
{
    public partial class MyFristGUI : Form
    {
        public MyFristGUI()
        {
            InitializeComponent();

            t.Interval = 4;
            flag = new Bitmap(480, 160);
            g = Graphics.FromImage(flag);
            t.Tick += T_Tick;

        }
        Bitmap flag;
        Graphics g;
        int x = 20;
        int y = 20;
        Timer t = new Timer();
       

        private void button1_Click(object sender, EventArgs e)
        {
            this.richTextBox1.Text = "button pressed";
        }

     
        private void T_Tick(object sender, EventArgs e)
        {    
            var rand2 = new Random();
            g = Graphics.FromImage(flag);
            g.Clear(Color.White);
            g.DrawCircle(new Pen(Brushes.Red, 2), x, y, 20);
            g.FillCircle(Brushes.Red, x, y, 20);
           
            x = x+rand2.Next(-10, 10);
            y = y+ rand2.Next(-10,10);
            if (x > 460) x = x - 10;
            if (x < 20) x = x + 10;
            if (y > 140) y = y - 10;
            if (y < 20) y = y + 10;
            pictureBox2.Image = flag;

         
           
        }



        private void MyFristGUI_Load(object sender, EventArgs e)
        {


        }

        private void button2_MouseClick(object sender, MouseEventArgs e)
        {
            
            var rand = new Random();
            x = rand.Next(20, 460);
            y = rand.Next(20, 140);
            t.Start();
            this.button2.Visible = false;
            this.button3.Text = "STOP";
            this.button3.Visible = true;
        }

        private void button3_MouseClick(object sender, MouseEventArgs e)
        {
            t.Stop();
            g.Clear(BackColor);
            this.button3.Visible = false;
            this.button2.Visible = true;
            pictureBox2.Image = flag;
        }

       
    }
}
